components/Snackbar/src/MDCSnackbarManager.h

```diff
@@ -77,6 +78,15 @@
  */
 + (void)setBottomOffset:(CGFloat)offset;
 
+/**
+ Determines the snackbar alignment to the screen.
+
+ If called within an animation block, the change will be animated.
+
+ @note This method must be called from the main thread.
```

Comment:
Nit: You can drop the main thread `@note` since this is a UIView, clients are expected to do all UI-related work on the main thread.